src/main/resources/db/changelog/changelog.sql

```diff
@@ -267,3 +267,46 @@ CREATE INDEX `Accounts-SynapseUserId-Index` ON `Accounts`(`synapseUserId`);
 ALTER TABLE `AccountRoles`
 MODIFY COLUMN `role` enum('DEVELOPER','RESEARCHER','ADMIN','TEST_USERS','WORKER','SUPERADMIN') NOT NULL;
 
+-- changeset bridge:13
+
+CREATE TABLE `Assessments` (
+  `guid` varchar(255) NOT NULL,
+  `appId` varchar(255) NOT NULL,
+  `identifier` varchar(255) NOT NULL,
+  `revision` int(10) unsigned NOT NULL,
+  `ownerId` varchar(255),
```

Comment:
A valid Assessment contains an ownerId. Shouldn't this also have a NOT NULL constraint?

Reply:
Yes I think at this point, it can never be left out. changed schema.